src/processTargets/modifiers/BoundedNonWhitespaceStage.ts

```diff
@@ -25,7 +18,7 @@ export type BoundedNonWhitespaceSequenceModifier = (
 export default class BoundedNonWhitespaceSequenceStage
   implements ModifierStage
 {
-  constructor(private modifier: BoundedNonWhitespaceSequenceModifier) {}
+  constructor(private modifier: ContainingScopeModifier | EveryScopeModifier) {}
```

Comment:
How come you weakened this type?

Reply:
Because there is really no gain. In every other modifier we are not this specific. Look at line for example.
https://github.com/pokey/cursorless-vscode/blob/6c54a17b9439917e890090f8eea7e3d9f2456ba3/src/processTargets/modifiers/scopeTypeStages/LineStage.ts#L12

Reply:
I would argue that we should be more strongly typed there as well.  I don't think you need to go and add it everywhere, but I don't think we should be removing it.  Time and time again we've found that the more you give Typescript the more it gives you.  I'm not in love with the changes in this PR that subvert and weaken our types

Reply:
Since the regex class was the only one that did this strongly typed and it created a lot of additional boilerplate I prefer to not use it

Reply:
Fine let's leave it. We can revisit later